recipe/meta.yaml

```diff
@@ -49,11 +49,25 @@ requirements:
 test:
   imports:
     - pylsp
+    - pylsp.config
+    - pylsp.plugins
   requires:
     - pip
+    - pytest
+    - numpy
+    - pandas
+    - matplotlib
+    - pyqt >=5,<6  # [not s390x]
+    - flaky
+  source_files:
+    - .pylintrc
```

Comment:
I don't think we need to lint this

Reply:
Unfortunately, there are tests that fail based on linting output. We can omit those tests or include their lint config in the test environment.

Reply:
ok, seems fine